aws/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
-name: kyverno-chart
-version:  v1.12.0-04-aws
+name: kyverno-aws-chart
+version:  v1.12.1
```

Comment:
There are a couple of places in scripts folder with this version. Need to fix that too.

Reply:
Got it.